vlib/vweb/README.md

```diff
@@ -61,3 +61,77 @@ That means that the template automatically has access to that action's entire en
 ### Deploying vweb apps
 
 Everything, including HTML templates, is in one binary file. That's all you need to deploy.
+
+## Getting Started
+
+To start with vweb, you have to import the module `vweb` first.
+After it, it's recommended to define a struct to access `vweb.Context`.
```

Comment:
After the import, define a struct to hold the `vweb.Context` and any other variables your program will need.  For this example, we will only keep the `vweb.Context` variable.

Reply:
Thank you for your updates. I would suggest to keep it as it is since it's just meant as a little info. There is a lot of room for improvements.
For the future it's better to make a fine structured separate documentation inside the `doc` folder.

Reply:
Even if you don't mention other variables, I would still change the first line.  Perhaps to this

After the import, define a struct to hold `vweb.Context`.